src/test/java/org/apache/sling/jcr/repoinit/PrincipalBasedAclTest.java

```diff
@@ -588,17 +600,17 @@ public void  principalAclNotAvailableNonExistingNode() throws Exception {
             U.parseAndExecute("create service user otherSystemPrincipal");
 
             // setting up principal-acl will not succeed (principal not located below supported path)
-            // but since the target node does not exist we cannot verify if an equivalent resource-based ac-setup exists
-            // (AccessControlManager.getPolicies would fail with PathNotFoundException) => relaxed behavior (SLING-9412)
+           
             String setup = "set principal ACL for otherSystemPrincipal \n"
                     + "allow jcr:read on /non/existing/path\n"
                     + "end";
-            U.parseAndExecute(setup);
-
-            Principal principal = adminSession.getUserManager().getAuthorizable("otherSystemPrincipal").getPrincipal();
-            for (AccessControlPolicy policy : acMgr.getPolicies(principal)) {
-                assertFalse(policy instanceof PrincipalAccessControlList);
+            try {
+                U.parseAndExecute(setup);
+                fail("Setting a principal ACL outside a supported path must not succeed");
+            } catch (RuntimeException e) {
+                // expected
```

Comment:
see above

Reply:
done in https://github.com/apache/sling-org-apache-sling-jcr-repoinit/pull/14/commits/cfadffb925ae3da29838a5f7a6a15630a147fbec